webapi/src/main/scala/org/knora/webapi/responders/ResponderV2.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2015 Lukas Rosenthaler, Benjamin Geer, Ivan Subotic,
+ * Tobias Schweizer, André Kilchenmann, and Sepideh Alassi.
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.responders
+
+import akka.pattern._
+import org.knora.webapi.IRI
+import org.knora.webapi.messages.v1.responder.standoffmessages.{GetMappingRequestV1, GetMappingResponseV1, GetXSLTransformationRequestV1, GetXSLTransformationResponseV1}
+import org.knora.webapi.messages.v1.responder.usermessages.UserProfileV1
+import org.knora.webapi.util.ConstructResponseUtilV2
+import org.knora.webapi.util.ConstructResponseUtilV2.{MappingAndXSLTransformation, ResourceWithValueRdfData}
+
+import scala.concurrent.Future
+
+/**
+  * An abstract class with utility methods for v2 responders.
+  */
+abstract class ResponderV2 extends Responder {
```

Comment:
Please rename `ResponderV2` to something more specific, like `ResourceTextResponder`, and have only the responders that really need `getMappingsFromQueryResultsSeparated` subclass it.

Reply:
fixed in https://github.com/dhlab-basel/Knora/pull/620/commits/0c13561911483771d29e52f822d66d117c03a4b4